src/pages/Map.js

```diff
@@ -9,31 +14,84 @@ const containerStyle = {
   top: '50.4px'
 };
 
-const center = {
-  lat: 50,
-  lng: 15,
-};
+const markers = [
+  {
+    id: 1,
+    subjectName: "Person 1",
+    location: "Poland",
+    date: "October 1, 1942",
+    position: { lat: 50.02, lng: 19.11 }
+  },
+  {
+    id: 2,
+    subjectName: "Person 2",
+    location: "Poland",
+    date: "September 12, 1933",
+    position: { lat: 50.04, lng: 19.57 }
+  },
+  {
+    id: 3,
+    subjectName: "Person 3",
+    location: "Poland",
+    date: "May 5, 1937",
+    position: { lat: 50.17, lng: 19.05 }
+  },
+  {
+    id: 4,
+    subjectName: "Person 4",
+    location: "Poland",
+    date: "September 19, 1934",
+    position: { lat: 52.13, lng: 21.0 }
+  }
+];
 
-const options = {
-  styles: mapStyles,
-};
 
 function Map() {
+  const [activeMarker, setActiveMarker] = useState(null);
   const { isLoaded, loadError } = useLoadScript({
     googleMapsApiKey: "AIzaSyBQ6AXhN1dWqYH5pjf8zuIoUyfTb1j0bAY",
   });
 
   if (loadError) return 'Error loading maps';
   if (!isLoaded) return 'Loading maps';
+  
+  const handleActiveMarker = (marker) => {
+    if (marker === activeMarker) {
+      return;
+    }
+    setActiveMarker(marker);
+  };
 
   return (
     <div className="Map">
       <GoogleMap
         mapContainerStyle={containerStyle}
-        center={center}
+        center={{ lat: 50, lng: 15 }}
```

Comment:
again, maybe we can use a constants file for this? what is 50' 15''

Reply:
50' 15'' centers the map over europe to start. would it be preferable to add it to styling/Constants.js? we don't really reuse this value though

Reply:
Yes, I think you're right! In that case, don't add it to constants but add a comment like //50' 15'' centers the map over europe to start so that people reading the code (i.e. other developers who take on our project can easily understand)